### ISSUE TITLE
Message: element click intercepted:

### ISSUE THREAD

**Issue by ZakatZakat** [mdichtler/PancakeSwap-Price-Scraper#3]
Hi. Trying to launch the script on default settings but got this error: 

ElementClickInterceptedException: Message: element click intercepted: Element <button class="sc-hKwCoD iSAXaV sc-bSqbRM dlCkVf open-currency-select-button" scale="sm">...</button> is not clickable at point (740, 481). Other element would receive the click: <div role="presentation" class="sc-jQrCkL jAarrT"></div>
  (Session info: chrome=92.0.4515.107)

I guess the website structure of pancake swap has changed that's why selenium can't click on things properly. Is it possible to fix it?

**Comment by ZakatZakat** [comment]
Update: 
I'have found out that the "token_search.send_keys(Keys.ENTER)" command is not working - selenium can't close the exchange window when token name is selected

**Comment by ktsi** [comment]
> Hi. Trying to launch the script on default settings but got this error:
> 
> ElementClickInterceptedException: Message: element click intercepted: Element ... is not clickable at point (740, 481). Other element would receive the click:
> 
> 
> (Session info: chrome=92.0.4515.107)
> I guess the website structure of pancake swap has changed that's why selenium can't click on things properly. Is it possible to fix it?

Yes the website structure is different now. Also some find_element_by_ is deprecated. I think that this code needs a revision which i am not skilled to manage.
